resources/kms-aliases.go

```diff
@@ -12,16 +13,22 @@ type KMSAlias struct {
 	name string
 }
 
-func (n *KMSNuke) ListAliases() ([]Resource, error) {
-	resp, err := n.Service.ListAliases(nil)
+func init() {
+	register("KMSAliase", ListKMSAliases)
```

Comment:
i dont think that `Aliase` is incorrect :)

Reply:
fixed